client/src/components/Profile/Profile.js

```diff
@@ -0,0 +1,306 @@
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        coverage: true,
+        shiftID: currentClickedID,
+        sentNotes: notes,
+      }),
+    })
+      .then((response) => {
+        return response.json();
+      })
+      .then((jsonResponse) => {
+        let newShifts = this.state.shifts;
+        for (let i = 0; i < newShifts.length; i++) {
+          if (newShifts[i].id == currentClickedID) {
+            newShifts[i].color = "#C187D3";
+          }
+        }
+        this.setState({ shifts: newShifts });
+      });
+    this.closeModal();
+  }
+  cancelClick() {
+    this.closeModal();
+  }
+  render() {
+    let timeslots = [];
+    const hours = [
+      "12am",
+      "1am",
+      "2am",
+      "3am",
+      "4am",
+      "5am",
+      "6am",
+      "7am",
+      "8am",
+      "9am",
+      "10am",
+      "11am",
+      "12pm",
+      "1pm",
+      "2pm",
+      "3pm",
+      "4pm",
+      "5pm",
+      "6pm",
+      "7pm",
+      "8pm",
+      "9pm",
+      "10pm",
+      "11pm",
+    ];
+
+    /*Every 8th element should be an "item-hours" header,
+      while every 1-7th element should be a shift cell.
+    */
+    for (var i = 0, ti = 0; i < 192; i += 1) {
+      if (i % 8 == 0) {
+        timeslots.push(<div className="item-hours">{hours[i / 8]}</div>);
+      } else {
+        timeslots.push(
+          <Timeslot
+            color={this.state.shifts[ti].color}
+            id={this.state.shifts[ti].id}
+            onClick={this.stateFixer}
+          />
+        );
+        ti += 1;
+      }
+    }
+
+    let wkdays = [];
+    for (var i = 0; i < 7; i += 1) {
+      wkdays.push(
+        <div className="item-wday">
+          {format(addDays(startOfWeek(currentDate), i), "dd MM/DD")}
+        </div>
+      );
+    }
+
+    const customStyles = {
+      content: {
+        top: "400px",
+        left: "50%",
+        width: "450px",
+        height: "400px",
+        transform: "translate(-50%, -50%)",
+        overflow: 0,
+      },
+    };
+
+    return (
+      <div>
+        <div>
+          <div className="everything">
+            <div className="myProfile">
+              <h1 className="myProfileText">My Profile</h1>
+            </div>
+            <div className="profileBox">
+              <div className="myProfileContainer">
+                <div className="profilePictureContainer">
+                  <div className="profilePicture"></div>
+                </div>
+
+                <div className="nameAndEmail">
+                  <div className="nameText">Bianca Lee</div>
+                  <div className="emailText">biancalee@berkeley.edu</div>
+                </div>
+
+                <div className="trainingLevels">
+                  <h1 className="trainingLevelsText">Training Levels</h1>
+                </div>
+                <div className="trainingLevelsContainer">
+                  <div className="trainingLevelsMoffittThird">
```

Comment:
Add css to vertically center the training level text and stars in each training level tag.